packages/react/src/ActionList/Item.tsx

```diff
@@ -455,7 +455,7 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
               {slots.blockDescription}
             </Box>
           </ItemWrapper>
-          {!inactive && Boolean(slots.trailingAction) && !container && slots.trailingAction}
+          {!inactive && Boolean(slots.trailingAction) && slots.trailingAction}
```

Comment:
We'll need to add a check like `!container`. If we go [the variable route](https://github.com/primer/react/pull/4697/files#r1652856505), we can do the same check (e.g. `!menuContext`)

Reply:
```suggestion
          {!inactive && !menuContext && Boolean(slots.trailingAction) && slots.trailingAction}
```